src/passes/Precompute.cpp

```diff
@@ -35,6 +35,8 @@ typedef std::unordered_map<GetLocal*, Literal> GetValues;
 
 // Execute an expression by itself. Errors if we hit anything we need anything not in the expression itself standalone.
 class StandaloneExpressionRunner : public ExpressionRunner<StandaloneExpressionRunner> {
```

Comment:
The name of the class, and comment above, are outdated with this change. How about `PrecomputingExpressionRunner`?